lldb/cmake/modules/AddLLDB.cmake

```diff
@@ -172,25 +172,29 @@ function(add_properties_for_swift_modules target reldir)
         target_link_directories(${target} PRIVATE
             "${CMAKE_OSX_SYSROOT}/usr/lib/swift"
             "${LLDB_SWIFT_LIBS}/macosx")
-	set(SWIFT_RPATH "/usr/lib/swift")
+	set(SWIFT_BUILD_RPATH "/usr/lib/swift")
+	set(SWIFT_INSTALL_RPATH "/usr/lib/swift")
       elseif(APSM_BOOTSTRAPPING_MODE STREQUAL "BOOTSTRAPPING")
         target_link_directories(${target} PRIVATE "${LLDB_SWIFT_LIBS}/macosx")
-	set(SWIFT_RPATH "${LLDB_SWIFT_LIBS}/macosx")
+	set(SWIFT_BUILD_RPATH "${LLDB_SWIFT_LIBS}/macosx")
+	set(SWIFT_INSTALL_RPATH "${LLDB_SWIFT_LIBS}/macosx")
```

Comment:
I don't know if this rpath needs to be set for both the build and install on macOS, but just kept it the same for now.

Reply:
Isn't this a change for macOS?

Reply:
It shouldn't be. Currently, `SWIFT_RPATH` is set as both the build and install rpath for macOS below, whereas I simply use two variables set to the same rpath on macOS after this pull, ie the end result should be the same.

I only had to copy these for macOS because on linux I need to set different build and install rpaths.